code/drasil-build/Build/Drasil/Make/MakeString.hs

```diff
@@ -39,15 +39,15 @@ renderVar f (Os nm _ _ _) = f nm
 renderVar f (Implicit nm) = "\"" ++ f nm ++ "\""
 renderVar f (Free nm) = f nm
 
--- | Constructor for converting a String into a MakeString.
+-- | Constructor for converting a 'String' into a 'MakeString'.
 makeS :: String -> MakeString
 makeS = Mr
 
--- | Constructor for Windows OS variables.
+-- | Constructor for Windows 'Os' variables.
```

Comment:
No, here OS is short for Operating System, so having both capitalized was correct (here and elsewhere).